### DISCUSSION TITLE
please obs error qt platfrom plugin how solve?

### DISCUSSION THREAD

**Discussion by arifAlzain64** [obsproject/obs-studio#7295] [Development Questions] [answered]
hello Tim obs
I can't use obs in windows 8.1, and the application not working
notification obs error qt platform plugin[](url)
![obs error](https://user-images.githubusercontent.com/75404299/188342858-33967040-cfff-436c-ae03-84be4a18453d.jpg)
how solved

**Comment by Mister36MN** [comment]
You may solve this by upgrading to an in-support version of Windows, namely recent Windows 10 versions or Windows 11, as OBS 28 is no longer supported in anything older on Windows.

In the future, please keep support questions in the discord or forums, as this space is for development discussion.

**Reply by Lolbit723** [reply]
Windows 8.1 is still in support tho? This comment is pretty dumb ngl

**Reply by Mister36MN** [reply]
"This is expected. OBS Studio 28 uses Qt 6, and Qt 6 does not support Windows 7, 8, or 8.1."

"support for Windows 8.1 will end on January 10, 2023." (From Windows).

Windows 8.1 is an extremely tiny amount of total windows users, and is going out of support in mere months. There is no functional reason to support it, so qt, which OBS is dependent on, has made that decision. Software should more often deprecate out-of-band or soon-to-be OOB systems before they end support, rather than after, like OBS has now done.

**Comment by RytoEX** [comment, marked as answer]
This is expected. OBS Studio 28 uses Qt 6, and Qt 6 does not support Windows 7, 8, or 8.1. Please upgrade your operating system to Windows 10 or newer, or please feel free to continue using OBS Studio 27.2 or older. Thank you for your understanding.

> 28.0: Updating to Qt 6
> 
> Qt 6 has dropped support for Windows 7 & 8, macOS 10.13 & 10.14, Ubuntu 18.04 and all 32-bit operating systems. As such, OBS will no longer be supported on these platforms.
> 
> For more details, please refer to our Discussion Post: https://github.com/obsproject/obs-studio/discussions/6481.
